macros/SpecName.ejs

```diff
@@ -1151,9 +1139,9 @@ var specList = {
         name : 'WebVTT: The Web Video Text Tracks Format',
         url  : 'https://w3c.github.io/webvtt/'
     },
-    'Web Workers':{ /* This draft has been merged into HTML WHATWG */
-        name : 'Web Workers',
-        url  : 'https://w3c.github.io/workers/'
+    'Web Workers':{
```

Comment:
I got a "sorry!" message on the following pages:

- https://developer.mozilla.org/en-US/docs/Web/API/Worker/onmessage
- https://developer.mozilla.org/en-US/docs/Web/API/AbstractWorker/onerror
- https://developer.mozilla.org/en-US/docs/Web/API/WorkerNavigator
- https://developer.mozilla.org/en-US/docs/Web/API/SharedWorker/port
- https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/location
- https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/importScripts
- https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/onerror
- https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/onoffline
- https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/ononline
- https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/onlanguagechange
- https://developer.mozilla.org/en-US/docs/Web/API/DedicatedWorkerGlobalScope/onmessage
- https://developer.mozilla.org/en-US/docs/Web/API/SharedWorkerGlobalScope

at this point I gave up on further changes as it's clear there's something very wrong with MDN, at least with regard to web worker pages.

What can we do in the meantime?